Controller/DemoController.php

```diff
@@ -21,6 +21,134 @@
 class DemoController extends Controller
 {
     /**
+     * Displays the sub folder if it exists
+     *
+     * @param int $locationId of a folder
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function showSubFolderLinkAction( $locationId )
+    {
+        $response = new Response();
+
+        $response->setSharedMaxAge( $this->getConfigResolver()->getParameter( 'content.default_ttl' ) );
```

Comment:
by default symfony makes this responses private, so dunno if we should add ```$response->setPublic()``` too. 
That said, dunno if the idea here was avoid cache. If not, shouldn't we set headers and Vary too?
https://github.com/ezsystems/DemoBundle/blob/master/Controller/DemoController.php#L80